packages/create-llama/helpers/index.ts

```diff
@@ -174,18 +188,28 @@ export const installTemplate = async (
       vectorDb: props.vectorDb,
       model: props.model,
       framework: props.framework,
+      dataSource: props.dataSource,
     });
 
-    // Copy test pdf file
-    await copyTestData(
-      props.root,
-      props.framework,
-      props.packageManager,
-      props.engine,
-      props.openAiKey,
-      props.vectorDb,
-      props.contextFile,
-    );
+    if (props.engine === "context") {
+      if (props.dataSource?.type === "file") {
+        if ("contextFile" in props.dataSource.config) {
+          await copyContextData(
+            props.root,
+            props.dataSource.config.contextFile,
+          );
+        } else {
+          await copyContextData(props.root);
+        }
+      }
+      await installDependencies(
```

Comment:
We missed calling `await` for installDependencies that why the error there were not caught.

Reply:
that's a nasty bug, can you make a new PR adding https://eslint.org/docs/latest/rules/require-await to eslint to avoid this in the future